src/logicblocks/event/processing/broker/broker/in_memory.py

```diff
@@ -1,4 +1,6 @@
-from ....store import InMemoryEventStorageAdapter
+from typing import overload
+
+from ....store import EventStore, InMemoryEventStorageAdapter
```

Comment:
This should import from `logicblocks.event.store` as it steps outside of the `processing` "package".